.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# See: https://help.github.com/articles/about-codeowners/
+
+# These owners will be the default owners for everything in the repo.
+* @Nuvindu @shafreenAnfar @NipunaRanasinghe
```

Comment:
Is this order correct ? AFAIR this should be something like this;

<module-owner> <team-lead> <library-team-overall-lead>